### PR TITLE
Update interfaces.md

### PR DESCRIPTION
**Pull request by absundr**
## Summary

Fixes a typo in the documentation

### CODE REVIEW

File: docs/csharp/fundamentals/types/interfaces.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Interfaces can contain instance methods, properties, events, indexers, or any co
 
 To implement an interface member, the corresponding member of the implementing class must be public, non-static, and have the same name and signature as the interface member.
 
-When a class or struct implements an interface, the class or struct must provide an implementation for all of the members that the interface declares but doesn't provide a default implementation for. However, if a base class implements an interface, any class that's derived from the base class inherits that implementation.
+When a class or struct implements an interface, the class or struct must provide an implementation for all of the members that the interface declares but doesn't provide a default implementation for the interface. However, if a base class implements an interface, any class that's derived from the base class inherits that implementation.
```
Comment (BillWagner) [inline docs/csharp/fundamentals/types/interfaces.md:31]: I agree that the original language is awkward. I'm not sure this helps. How about the following wording:

```suggestion
A class or struct that implements an interface must provide an implementation for all declared members without a default implementation provide by the interface. However, if a base class implements an interface, any class that's derived from the base class inherits that implementation.
```

**Review by BillWagner — COMMENTED** [review]
Thanks for making this suggestion @abhijithsundr 

I proposed an alternative wording for the change. Let me know what you think and then I'll merge this.

File: docs/csharp/fundamentals/types/interfaces.md (hunk 1 above)
Comment (abhijithsundr) [reply docs/csharp/fundamentals/types/interfaces.md]: Great, this works.

Comment (abhijithsundr) [reply docs/csharp/fundamentals/types/interfaces.md]: Thank you

**Review by BillWagner — APPROVED** [review]
Thanks again @abhijithsundr 

I'll :shipit: now.

**Pull request merged**